<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>265</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>痒痒鼠助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="first_page">
       <attribute name="title">
        <string>主页面</string>
       </attribute>
       <widget class="QPushButton" name="btn_move_left">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>左边</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="resolution_label">
           <property name="text">
            <string>当前屏幕分辨率未知</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_resolution">
           <property name="text">
            <string>分辨率检测</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="second_page">
       <attribute name="title">
        <string>魂十</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="third_page">
       <attribute name="title">
        <string>魂土</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="fourth_page">
       <attribute name="title">
        <string>突破</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
